lib/main.dart

```diff
@@ -129,8 +129,8 @@ class _MyAppState extends State<MyApp> {
   }
 
   Future<SMKitWorkout> getDemoWorkout() async{
-    var introURL = await getFileUrl("customWorkoutIntro.mp3");
-    var highKneesIntroURL =  await getFileUrl("HighKneesSound.mp3");
+    var introURL = await getFileUrl("customWorkoutIntro.mp3"); // local sound url
```

Comment:
@SencyNetanel my intention was to keep two examples - one for local url, and another one for remote.
not replacing the exiting example for local with remote example

Reply:
There is 2 exemples intro url is local and highKneesIntroURL is remote do you mean you want two functions that do the same thing just one get's highKneesIntroURL from local and the other dosn't?